cabal.project

```diff
@@ -0,0 +1,5 @@
+packages:
+  .
+
+allow-newer:
+  aeson-better-errors:aeson
```

Comment:
can be removed if this is merged and new version released: https://github.com/hdgarrood/aeson-better-errors/pull/24